faker/tests/requirements.txt

```diff
@@ -0,0 +1 @@
+mock==1.0.1
```

Comment:
I used `mock` for the test I added, but I'm not sure how y'all would like to specify this requirement. It only needs to be installed for python < 3.3.

Adding a requirement here means in order to run tests, you'd need to run
```
$ pip install -r tests/requirements.txt
```
This is how django handles this problem: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/unit-tests/#running-unit-tests-dependencies

Is there another way you'd like to handle this? If we stick with this route, I should probably add the requirement to the documentation somewhere, but I wanted to run this by y'all first to see if this is how you'd like to handle it.